src/commands/synthetics/run-tests-command.ts

```diff
@@ -161,15 +162,20 @@ export class RunTestsCommand extends Command {
       })
     )
 
-    // Pass root polling timeout to global override to get it applied to all tests if not defined individually
-    this.config.global.pollingTimeout = this.config.global.pollingTimeout ?? this.config.pollingTimeout
+    let pollingTimeoutCliArgument
+    try {
+      pollingTimeoutCliArgument = parseOptionalInteger(this.pollingTimeout)
+    } catch (error) {
+      throw new CiError('INVALID_CONFIG', `Invalid value for \`pollingTimeout\`: ${error.message}`)
+    }
```

Comment:
Instead of doing the validation manually, could we follow the `clipanion` example of a `validator`? https://mael.dev/clipanion/docs/validation#validating-options

Reply:
Wow, I haven't even taken the time to look at the docs... shame on me 😄

Reply:
It requires a new dependency (`typanion`) though, but it may be a good thing for the repo.

Reply:
I think it's acceptable - seems lightweight and helpful